src/components/member-profile/contribution-type/index.js

```diff
@@ -27,11 +27,25 @@ const renderActiveTasks = (tasks) => {
 
 const ContributionType = (props) => {
   const { fullName, type, imageLink, contributions, devUser, tasks } = props;
+  const [showMoreContent, setShowMoreContent] = useState(false);
 
-  const [showMoreContent, setShowMoreContent] = useState(true);
+  useEffect(() => {
+    if (type === 'Noteworthy' || type === 'All') {
+      setShowMoreContent(contributions.length > 0);
+      return;
+    }
+    setShowMoreContent(tasks.length > 0);
+  }, [tasks.length, contributions.length]);
 
   const showMoreContentHandler = () => {
-    setShowMoreContent((prevstate) => !prevstate);
+    if (type === 'Noteworthy' || type === 'All') {
```

Comment:
can we create a state called like following and use it everywhere it is needed:
```js
const isContribution = type === 'Noteworthy' || type === 'All'
```

Reply:
Yeah it would be better i will add it